proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,535 @@
+   DJing zu beginnen und ermöglicht Softwareentwickler, Implementierungsdetails
+   zu überprüfen und Änderungen einzubringen.
+2. Der Satzungszweck wird insbesondere verwirklicht durch:
+   1. die Verbreitung von Mixxx als freie und quelloffene Software für Bildung,
+      Kunst und kulturelle Zwecke
+   2. das Erstellen von Handbüchern und Anleitungen, um das Lernen und die
+      Entwicklung zu unterstützen
+   3. die Organisation von Hilfe für Anwender und Bildungsprogrammen
+   4. die Begleitung von Studenten während Praktika in der Softwareentwicklung
+   5. den Aufbau einer inklusiven und vielfältigen Gemeinschaft, in der
+      Mitwirkende aller Hintergründe, Geschlechter und Fähigkeitsstufen
+      willkommen sind.
+   6. die Aufrechterhaltung der Entwicklungsinfrastruktur für niedrige
+      Einstiegshürden
+   7. die Unterstützung der Entwicklung von Mixxx, ohne Kontrolle über Details
+      auszuüben, die unabhänging von den Regeln und Zweckes des Vereins sind.
+   8. die Unterstützung der Behörden mit Fachwissen über die DJ-Hardware und
+      Software (z.B. bei Wettbewerbsuntersuchungen)
+3. Eine Änderung des Vereinszwecks darf nur im Rahmen des in §3 (1) gegebenen
+   Rahmens erfolgen.
+
+## 3. Gemeinnützigkeit
+
+1. Der Verein verfolgt im Rahmen seiner Tätigkeit gemäß § 2 der Satzung
+   ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnittes
+   steuerbegünstigte Zwecke der Abgabenordnung (§§ 51ff. AO). Der Verein ist
+   politisch, ethnisch und konfessionell neutral. Er ist selbstlos
+   tätig und verfolgt nicht in erster Linie eigenwirtschaftliche Zwecke.
+2. Die Mittel des Vereins sind ausschließlich zu satzungsgemäßen Zwecken zu
+   verwenden. Die Mitglieder erhalten ausschließlich Erstattungen entstandener
+   Kosten, aber keine direkten Zuwendungen aus Mitteln des Verein.
+3. Niemand darf durch Vereinsausgaben, die dem Vereinszweck fremd sind oder
+   durch unverhältnismäßig hohe Vergütungen begünstigt werden.
+
+## 4. Arten der Mitgliedschaft und Mitglieder des Vereins
+
+1. Mitglieder des Vereins können natürliche und juristische Personen werden,
+   die die Ziele des Vereins mittragen und unterstützen wollen. Folgende Arten
+   von Mitgliedschaften sind möglich:
+   1. **Kernmitglieder**: Natürliche Personen, die den Vereinszweck und dessen
+      Ziele aktiv unterstützen. Von ihnen wird in erster Linie erwartet, dass
+      sie mitarbeiten und einen Beitrag leisten, an der Mitgliederversammlungen
+      teilnehmen und ihr Stimmrecht ausüben.
+   2. **Außerordentliche Mitglieder**: Natürliche und juristische Personen,
+      die Vereinsziele unterstützen, aber kein Stimmrecht und keine
+      Mitwirkungspflicht haben. Dies gibt Mitgliedern die Möglichkeit,
+      ihre Mitarbeit und Beiträge zu Mixxx zu pausieren, ohne zwangsläufig
+      ihre Mitgliedschaft zu verlieren.
+   3. **Fördernde Mitglieder**: Außerordentliche Mitglieder, die durch
```

Comment:
This is the KDE model. It would give a reliable long term support and is a base if we plan to employ people. 
I think at the moment it is useful to have this initially in our Satzung because a later change can be dificult and it also protects the core members from regular fees.

Reply:
@JoergAtGithub Is OK for you or do we have to adjust something?

Reply:
Why do have 1. and 3. the right to participate the "Mitgliederversammlung, but 2. not?

Reply:
Ah I see we just need to move the sentence to 2. Than it is valid for both.

Reply:
Now we have the sentence twice - not wrong, but redundant